### PR TITLE
Add Logitech Options 8.20.329

### PR DESCRIPTION
**Pull request by auanasgheps**
Please note this installer can't be made silent, it's custom and does not accept any silent switch install, therefore I don't know if can be accepted.

- [x] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [x] Have you validated your manifest locally with `winget validate <manifest>`, where `<manifest>` is the name of the manifest you're submitting?
- [x] Have you tested your manifest locally with `winget install -m <manifest>`?

-----

###### Microsoft Reviewers: [Open in CodeFlow](http://wpcp.azurewebsites.net/CodeFlowProtocolProxy2.php?pullrequest=https://github.com/microsoft/winget-pkgs/pull/3365)

### CODE REVIEW

File: manifests/Logitech/Options/8.20.329.yaml [hunk 1]
```diff
@@ -0,0 +1,18 @@
+Id: Logitech.LogitechOptions
```
Comment (OfficialEsco) [inline manifests/Logitech/Options/8.20.329.yaml:1]:
```suggestion
Id: Logitech.Options
```

**Review by OfficialEsco — CHANGES_REQUESTED** [review]
`Manifest: The Id property of a manifest must include the publisher name and application name separated by a period. actual: 'Logitech.LogitechOptions', expected 'Logitech.Options'`

**Comment by auanasgheps** [comment]
@OfficialEsco thanks for your help! It worked.

**Comment by KevinLaMS** [comment]
This pops a UI which prevents silent install.

**Comment by auanasgheps** [comment]
Please note this installer can't be made silent, it's custom and does not accept any silent switch install, therefore I don't know if can be accepted.

Chocolatey is using an autokey in order to accept prompts.

**Comment by auanasgheps** [comment]
Closing because has been added with #5695

**Pull request closed without merging**